test-configs.yaml

```diff
@@ -941,6 +948,9 @@ test_configs:
   - device_type: kirkwood_db_88f6282
     test_plans: [boot_nfs]
 
+  - device_type: meson_g12a_sei510
+    test_plans: [boot, kselftest, simple]
```

Comment:
Please drop kselftest for now.  This test plan is not currently useful on amlogic boards.